opencti-platform/opencti-graphql/src/schema/schema-validator.ts

```diff
@@ -71,7 +71,18 @@ export const validateAndFormatSchemaAttribute = (
           const patchedInstance = jsonpatch.applyPatch(R.clone(initial), patch).newDocument as any;
           validationValues = patchedInstance[editInput.key];
         }
-        const valid = validate(validationValues);
+
+        let valid = false;
+        if (attributeDefinition.multiple) {
+          // type=object && multiple -> it's an array than can be validated as-is
+          valid = validate(validationValues);
+        } else {
+          // object not multiple, only one value to check
+          if (validationValues.length > 1) {
+            throw ValidationError(attributeName, { message: 'Object is not multiple but has more than 1 value', data: { attribute: attributeName } });
+          }
+          valid = validate(validationValues[0]);
+        }
```

Comment:
the function `validateAndFormatSchemaAttribute` checks that the input provided is conform to the schema attribute definition. The input MUST be provided as an array, so when the value is unique, it's still turned into and array with `value: Array.isArray(v) ? v : [v]` (see `validateInputCreation`)

This case is here for when the type of attribute is `object`. There are very few cases of objects : TAI's `weight`, `height` ; and now the `user_confidence_level`.

Both `weight` and `height` attributes are actually **array** object, so the `multiple` flag is true.
`user_confidence_level` is a plain object, not an array so `multiple` is false.

➡️ We were not taking `multiple` into account to know if the values are actually to be validated as an array or not.